tests/CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ set(CLI11_MULTIONLY_TESTS TimerTest)
 find_package(Catch2 CONFIG)
 
 if(Catch2_FOUND)
-  if(NOT TARGET Catch2::Catch2)
+  if(NOT (TARGET Catch2::Catch2 OR TARGET Catch2::Catch2WithMain))
```

Comment:
Unrelated: I tried to build with Catch2 v3, hit this error. Then I hit another one and gave up (later it's trying to add_sources to catch_main whcih is just an ALIAS in the case it's v3).